roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: restart nginx
+  become: yes
+  service:
+    name: nginx
+    state: restarted
+
+- name: reload nginx
```

Comment:
I only found references for the `restart` handler. Is this `reload` handler meant to be used anywhere?